yarn.lock

```diff
@@ -3695,6 +3721,13 @@ read-pkg-up@^2.0.0:
     find-up "^2.0.0"
     read-pkg "^2.0.0"
 
+read-pkg-up@^3.0.0:
```

Comment:
after `yarn`.. 😂

![image](https://user-images.githubusercontent.com/8341033/34524441-7d3534d6-f0d6-11e7-876b-ff869f5701fd.png)

Reply:
Upgraded yarn - problem went away. Hopefully the last time 😭